dracut/30ignition/module-setup.sh

```diff
@@ -62,6 +68,10 @@ install() {
 
     # needed for openstack config drive support
     inst_rules 60-cdrom_id.rules
+
+    # clean up networking so when we switch to the real root,
+    # NetworkManager will set up the correct ignition-configured networking
+    # inst_hook cleanup 10 "$moddir/networking-cleanup"
```

Comment:
Any reason for keeping this?

Reply:
Nope, will remove, thanks for catching that!